hack/dind-cluster.sh

```diff
@@ -44,59 +43,58 @@
 # command would ensure configuration was stored at
 # /tmp/openshift-dind/cluster/my-cluster:
 #
-#    OS_INSTANCE_PREFIX=my-cluster hack/dind-cluster.sh [command]
+#    OPENSHIFT_INSTANCE_PREFIX=my-cluster hack/dind-cluster.sh [command]
 #
 # It is also possible to specify an entirely different configuration path:
 #
-#    OS_DIND_CONFIG_ROOT=[path] hack/dind-cluster.sh [command]
+#    OPENSHIFT_CONFIG_ROOT=[path] hack/dind-cluster.sh [command]
 #
-# Running Tests
-# -------------
-#
-# This script includes a shortcut for running the networking e2e
-# tests.  The test-net-e2e command will run the extended networking
-# tests against an already running dind cluster:
-#
-#     hack/dind-cluster.sh test-net-e2e
-#
-# Bash Aliases
-# ------------
-#
-# The following bash aliases are available in the cluster containers:
+# Suggested Workflow
+# ------------------
 #
-# oc-create-hello - create the 'hello' example app
-# oc-less-log - invoke 'less' on the openshift daemon log (will target
-#               the master or node log depending on the type of node)
-# oc-tail-log - invoke tail on the openshift daemon log
+# When making changes to the deployment of a dind cluster or making
+# breaking golang changes, the 'restart' command will ensure that an
+# existing cluster is cleaned up before deploying a new cluster.
 #
-# Process Management
-# ------------------
+# When only making non-breaking changes to golang code, the 'redeploy'
+# command avoids restarting the cluster.  'redeploy' rebuilds the
+# openshift binaries and deploys them to the existing cluster.
 #
-# Due to docker-in-docker conflicting with systemd when running in a
-# container, supervisord is used instead.  The 'supervisorctl' command
-# is the equivalent of 'systemctl' and logs for managed processes can
-# be found in /var/log/supervisor.
+# Running Tests
+# -------------
 #
-# Loopback Devices
-# ----------------
+# The extended tests can be run against a dind cluster as follows:
 #
-# Due to the way docker-in-docker daemons interact with loopback
-# devices, it is important to invoke 'dind-cluster.sh stop' on a
-# running cluster instead of manually stopping the containers.  This
-# ensures that the containerized docker daemons are gracefully
-# shutdown and allowed to release their loopback devices before
-# container shutdown.  If the daemons are not stopped before container
-# shutdown, the associated loopback devices will be effectively
-# unusable ('leaked') until a subsequent host reboot.  If enough
-# loopback devices are leaked, cluster boot may not be possible since
-# each openshift node running in a container depends on a docker
-# daemon requiring 2 loopback devices.
+#     OPENSHIFT_CONFIG_ROOT=dind test/extended/networking.sh
```

Comment:
I tried to run this after starting my dind cluster, but got:

+(test/extended/networking.sh:169): main(): case "${CONFIG_ROOT}" in
+(test/extended/networking.sh:175): main(): CONFIG_ROOT=/tmp/openshift-dind-cluster/openshift
+(test/extended/networking.sh:176): main(): '[' '!' -d /tmp/openshift-dind-cluster/openshift ']'
+(test/extended/networking.sh:177): main(): os::log::error 'OPENSHIFT_CONFIG_ROOT=dind but dind cluster not found'
+(test/extended/../../hack/util.sh:725): os::log::error(): os::log::with-severity 'OPENSHIFT_CONFIG_ROOT=dind but dind cluster not found' ERROR
+(test/extended/../../hack/util.sh:710): os::log::with-severity(): local 'msg=OPENSHIFT_CONFIG_ROOT=dind but dind cluster not found'